tests/v2/validation/rbac/rbac.go

```diff
@@ -257,3 +259,78 @@ func createRole(client *rancher.Client, context string, roleName string, rules [
 	return
 
 }
+
+func waitForActiveCluster(client *rancher.Client, clusterID string) error {
+	err := kwait.Poll(500*time.Millisecond, 30*time.Minute, func() (done bool, err error) {
+		client, err = client.ReLogin()
+		if err != nil {
+			return false, err
+		}
+		clusterResp, err := client.Management.Cluster.ByID(clusterID)
```

Comment:
Maybe use the Steve client instead, since that is what the UI uses for check cluster state

Reply:
Please correct me if I am wrong but Steve client won't work for RKE1 cluster?

Reply:
hmmmm ... I'm not sure would the UI use management.cattle.io on an RKE1 cluster?

Reply:
yes RKE1 cluster uses management.cattle.io